README.md

```diff
@@ -88,6 +88,22 @@ Note that the default `service-id` includes more than the container name (see be
 
 	docker run -d --name redis-1 -e SERVICE_ID=redis-1 -p 6379:6379 redis
 
+#### AWS SNS backend
+
+Publishes service registration events to AWS SNS. Each message will contain a custom `MessageAttribute` called `DOCKER.EVENT`, whose value will be one of:
+
+ * `PING`
```

Comment:
No particular reason -- I interpreted `Ping()` as a backend liveliness check rather than a service registration related method. Your call :)

Just realised I haven't included `SERVICE_REFRESH` in the list, will update.